.github/workflows/tests.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        tarantool-version: ["1.10", "2.8"]
+        tarantool-version: ["1.10", "2.8", "2.10"]
```

Comment:
I think it is enough to run the tests on LTS and the latest version of tarantool.

Reply:
I've replaced it by `["1.10", "2.10"]`.